WorkspaceServer/WorkspaceServer.csproj

```diff
@@ -107,6 +107,7 @@
     <ProjectReference Include="..\Microsoft.DotNet.Try.Project\Microsoft.DotNet.Try.Project.csproj" />
     <ProjectReference Include="..\MLS.Agent.Tools\MLS.Agent.Tools.csproj" />
     <ProjectReference Include="..\Microsoft.DotNet.Try.Protocol\Microsoft.DotNet.Try.Protocol.csproj" />
+    <ProjectReference Include="..\XPlot.DotNet.Interactive.KernelExtensions\XPlot.DotNet.Interactive.KernelExtensions.csproj" />
```

Comment:
Thoughts on putting this in MLS.Agent rather than here?

Reply:
What is the advantage of one over another ? Right now all the kernel specific stuff lives in WorkspaceServer right ?